draft-west-cookie-incrementalism.md

```diff
@@ -134,6 +136,13 @@ the near-term. User agents should:
 
     This is spelled out in more detail in {{require-secure}}.
 
+3. Ensure the scheme, as well as the registrable domain, of the
+   "site for cookies" and request match when making a same-site decision.
+   That is, "http://site.example" and "https://site.example" should be
+   considered cross-site
```

Comment:
Period at the end.

Reply:
Apologies for turning you into an editor, here and elsewhere.
Done